### ISSUE TITLE
Fix accessibility of preset tabs in banner/widget

### ISSUE THREAD

**Issue by sidvishnoi** [interledger/publisher-tools#305]
Problem: 
- Anything inside `role=tab` is marked as presentation
- So, a tab cannot contain the edit "button" as well as an input.
- So, screen reader users won't even know that there's a button inside the tab and that the preset name is editable.
- The current JS implementation for making the tab title editable is patchy as we well, and not accessible.

Tasks:
- Remove the edit button and input from `role=tab` and move it into the active `role=tabpanel`.
- Position the edit functionality such that visually it looks like we're editing the tab title.
- Add double click on tabpanel button to trigger same edit functionality, as an affordance for mouse users.
- Simplify edit logic while we're there.
